docs/conf/atlantic/2023/writing-day.rst

```diff
@@ -67,3 +67,20 @@ together and offer each other some feedback on the important documentation
 we're using to move our careers forward.
 
 Let's help each other get ready for the job fair!
+
+The Blog Writing Room: Blogging for technical writers and documentarians
```

Comment:
I'd recommend shortening the title to "The Blog Writing Room" - the audience is implied and you mention it in your second sentence.